react-components/src/analysis/view/AnalysesView.js

```diff
@@ -76,6 +78,16 @@ function AnalysisName(props) {
                 <sup>{name}</sup>
             </span>
         )
+    } else if (parentId) {
+        return (
+            <span title={formatMessage(intl, "goOutputFolderOf") + " " + name}
```

Comment:
Looks like `goOutputFolderOf` is no part of this PR, but for another PR, it would be better if this message in `react-components/src/analysis/messages.js` accepted `name` as a parameter:
```javascript
"goOutputFolderOf": "Go to output folder of {name}"
```